src/python_testing/TCP_Tests.py

```diff
@@ -152,10 +180,8 @@ async def test_TC_SC_8_7(self):
         asserts.assert_equal(device.isActiveSession, True, "Large Payload Session should be active over TCP connection")
         asserts.assert_equal(device.sessionAllowsLargePayload, True, "Session does not have associated TCP connection")
 
-        commands = Clusters.Objects.OnOff.Commands
         try:
-            await self.send_single_cmd(cmd=commands.Toggle(), endpoint=1,
-                                       payloadCapability=ChipDeviceCtrl.TransportPayloadCapability.MRP_OR_TCP_PAYLOAD)
+            self.send_arm_cmd()
```

Comment:
For this command, we need to pass MRP_OR_TCP_PAYLOAD.

Reply:
whoops, sorry, good catch. updated to parameterize.